### ISSUE TITLE
It's unable to press space in HTML input/textarea

### ISSUE THREAD

**Issue by ArtElect** [nhn/tui.image-editor#556]
**Describe the bug**
We cannot insert spaces in HTML input/textarea in the same page as Toast UI

**To Reproduce**

1. Go to 'http://jsfiddle.net/m3h2cvfz/40/'
2. Click on 'input area'
3. Type anything + space

**Expected behavior**
 Expect to insert space in any HTML textarea or input area correctly.

**Screenshots**
None.

**Desktop (please complete the following information):**

- OS: [Windows 10 Home]
- Browser [Chrome]
- Version [89]

**Smartphone (please complete the following information):**

None.

**Additional context**
Is there a way to disable the hand button?
Because everytime when I press the space it would press the hand button.

**Comment by lja1018** [comment]
@ArtElect 
Thank you for reporting. It seems to be the same issue as #555 and I will resolve with it quickly.

**Comment by lja1018** [comment]
@ArtElect 
Resolved in [v3.14.1](https://github.com/nhn/tui.image-editor/releases/tag/v3.14.1). Thank you.

**Comment by yashrahurikar23** [comment]
Tried installing the version, 3.14.1 it still exists

**Comment by ArtElect** [comment]
@yashrahurikar23 yeah, if you only press once it won't work but if you hold down the Space it will work.

**Comment by lja1018** [comment]
@yashrahurikar23 @ArtElect 
I have exactly identified the problem! I will fix it ASAP.

**Comment by yashrahurikar23** [comment]
Thanks @lja1018

**Comment by marcodafonseca** [comment]
When will a version be released with this fix in it? It is causing problems in my client's platform as we render the image editor in a modal. So once the modal is closed the spacebar issue is shown in all inputs across the web page. Needless to say this is quite a glaring problem for us

**Comment by lja1018** [comment]
@marcodafonseca 
It will probably be resolved tomorrow through the 3.14.2 release. Thank you!

**Comment by lja1018** [comment]
@yashrahurikar23 @ArtElect @marcodafonseca 
Resolved in [v3.14.2](https://github.com/nhn/tui.image-editor/releases/tag/v3.14.2). Thank you :)

**Comment by eomttt** [comment]
it also fire v3.15.2
when i use `addText` method.
textarea open and focusing but when i typing english or number and type `space` focus out
but korean is work very well (but i typing korean and click `space` twice same)

example
abc(space)<- focus out
123(space)<- focus out
가나다(space)라마바사<- work well
가나다(space)(space)<-focus out
